### ISSUE TITLE
Clearing marker list

### ISSUE THREAD

**Issue by Zartexo** [louisnw01/lightweight-charts-python#21]
Hi,

when setting new data to the chart the markers from before will still be in the array of markers and shown at the first data point.

![Screenshot 2023-06-11 122302](https://github.com/louisnw01/lightweight-charts-python/assets/104310309/b8e2ed72-6753-4a17-b4e5-9c5dfe7ffe21)

I suggest to add the function `remove_marker_all()` to clear the array of markers:

```
    def remove_marker_all(self):
        """
        Clears the array of markers.\n
        """
        self.run_script(f'''
           {self.id}.markers = []
           {self.id}.series.setMarkers({self.id}.markers)''')
```

BR

**Comment by louisnw01** [comment]
Hi!

This sounds good;  I'll add a `clear_markers` method for the next update, as well as a `clear_horizontal_lines` for consistency.

**Comment by louisnw01** [comment]
Hi,

This has been implemented in the latest version (1.0.13)

Thanks!
Louis
